# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.cpp
# Commit: 4e05361c6b08
# Full Hash: 4e05361c6b08aad43b939c33f5aa608a3d9ccbc8
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p5 - Move nsPrintData::mPreparingForPrint to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146977
# ==============================================================================

diff -r cc4de7b25404 -r 4e05361c6b08 layout/printing/nsPrintData.cpp
--- a/layout/printing/nsPrintData.cpp	Sun May 22 09:37:56 2022 +0000
+++ b/layout/printing/nsPrintData.cpp	Sun May 22 09:37:57 2022 +0000
@@ -24,11 +24,7 @@
 //-- nsPrintData Class Impl
 //---------------------------------------------------
 nsPrintData::nsPrintData(ePrintDataType aType)
-    : mType(aType),
-      mPrintDocList(0),
-      mOnStartSent(false),
-      mIsAborted(false),
-      mPreparingForPrint(false) {}
+    : mType(aType), mPrintDocList(0), mOnStartSent(false), mIsAborted(false) {}
 
 nsPrintData::~nsPrintData() {
   // Only Send an OnEndPrinting if we have started printing